documentation/Using-the-SDK/.attachments/dpp.svg

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@zachnew-msft I believe this was your diagram originally. What do you think of the proposed changes?

Reply:
Can I suggest flipping the whole diagram top to bottom since bottom to top is slightly illogical to follow when reading a document top to bottom (Here we would have CDS at the top and Composite Cooker at the bottom).

 Also for the "can send data to" where the data flow is optional, perhaps use dashed arrows. Also using smaller fonts for all text items inside of arrows for better formatting will make it more readable. An optional additional suggestion is to make a Composite cooker A and Composite cooker B to show that they can communicate with other CCs but this change isn't necessary.

Reply:
That sounds fine.